src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -73,6 +73,13 @@ public DisputesController(IBus bus, IMediator mediator, ILogger<DisputesControll
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public async Task<IActionResult> CreateAsync([FromBody] Models.Disputes.NoticeOfDispute dispute, CancellationToken cancellationToken)
     {
+        if (dispute.TicketNumber is not null && dispute.ViolationTicket is not null && dispute.ViolationTicket.TicketNumber is not null 
```

Comment:
Lets create a post go-live Jira to move this logic to the `NoticeOfDisputeValidator` and ensure this API is using the validator.

Reply:
Here is the Jira ticket I created for tracking this: [TCVP-2892](https://justice.gov.bc.ca/jira/browse/TCVP-2892)